README.md

```diff
@@ -110,17 +110,24 @@ forge script script/DeployNodlMigration.sol --zksync --rpc-url https://sepolia.e
 Afterwards the user you onboarded should be able to mint NFTs as usual via the `safeMint(ownerAddress, metadataUri)` function.
 
 ### Deploying MigrationNFT contract
-The `MigrationNFT` contract allows the minting of a reward NFT when users bridge enough tokens through `NODLMigration`. You will need to set the following environment variables:
+The `MigrationNFT` contract allows the minting of a reward soulbound NFT when users bridge enough tokens through `NODLMigration`. Users can "level up" depending on the amount of tokens they bring in, with each levels being a sorted list of bridged amounts.
+
+You will need to set the following environment variables:
 - `N_MIGRATION`: address of the `NODLMigration` contract
-- `N_MAX_NFTS`: maximum number of NFTs available
-- `N_MIN_AMOUNT`: minimum amount of tokens to bridge to be elligible for a NFT.
-- `N_TOKENS_URI`: URI of NFT metadata.
+- `N_MAX_HOLDERS`: maximum number of participants
+- `N_TOKENS_URI_ROOT`: URI of NFT metadata folder where each level metadata is later presented
+- `N_LEVELS`: number of levels to set
```

Comment:
What are these levels?

Reply:
Basically depending on how many tokens you bridge you get better looking NFTs. Idea from products.

Reply:
So perhaps it's better to be called `N_AMOUNT_REUQIRED_FOR_RANK_0` or `N_AMOUNTS_REQUIRED_FOR_RANKS`